spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkDistributionAndOrderingUtil.java

```diff
@@ -84,14 +85,14 @@ public static Distribution buildRequiredDistribution(Table table, DistributionMo
 
   public static Distribution buildCopyOnWriteDistribution(Table table, Command command,
                                                           DistributionMode distributionMode) {
-    if (command == DELETE) {
-      return buildCopyOnWriteDeleteDistribution(table, distributionMode);
+    if (command == DELETE || command == UPDATE) {
```

Comment:
I thought about this and I think we can reuse the same distribution as in DELETEs. A few edge cases is updating fields used in the partition spec and sort order. I think the current logic would probably behave reasonably (see the table below). Would appreciate any feedback.

Reply:
Yeah, after looking at this I think I agree.